pyproject.toml

```diff
@@ -1,45 +1,44 @@
 [project]
-name = "reichlab-python-template"
-description = "Python template for Reich Lab projects"
+name = "postpredict"
+description = "Postprocess probabilistic predictions"
 license = {text = "MIT License"}
 readme = "README.md"
-requires-python = '>=3.10'
+requires-python = '>=3.9'
 classifiers = [
     "Programming Language :: Python :: 3",
     "License :: OSI Approved :: MIT License",
 ]
 dynamic = ["version"]
 
 dependencies = [
-    "structlog",
-    "rich",
-    "toml"
 ]
 
 [project.optional-dependencies]
 dev = [
     "coverage",
-    "freezegun",
     "mypy",
     "pre-commit",
     "pytest",
     "ruff",
+    "toml",
     "types-toml",
 ]
 
-[project.entry-points."console_scripts"]
-reichlab_python_template = "reichlab_python_template.app:main"
+[project.urls]
+Repository = "https://github.com/reichlab/postprocess.git"
 
 [build-system]
 # Minimum requirements for the build system to execute.
 requires = ["setuptools", "wheel"]
 
-[tools.setuptools]
-packages = ["reichlab_python_template"]
+[tool.pytest.ini_options]
+tmp_path_retention_policy = "none"
+testpaths = [
```

Comment:
Looks like this broken the pytest incantation in the code checks GitHub action because there's no longer a folder called `tests`

Reply:
I saw these errors, but I was thinking I'd like to leave this setup in place, with plans to add tests as soon as i start adding code